docs/_blog/2019-10-30-a-technical-autopsy-of-a-containerized-node-js-dependency-insights-microservice-application.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "A technical autopsy of a containerized ‘Node.js dependency insights’ microservice application"
+
```

Comment:
Is there a reason a space is here? Will it cause a problem?

Reply:
I removed the space, no it won't cause a problem, but to be consistent I will remove it